saleor/order/models.py

```diff
@@ -329,8 +329,8 @@ class OrderEvent(models.Model):
     order = models.ForeignKey(
         Order, related_name='events', on_delete=models.CASCADE)
     parameters = JSONField(
-        blank=True, default={},
-        dump_kwargs={'cls': CustomJsonEncoder, 'separators': (',', ':')})
+        blank=True, default=dict,
+        encoder=CustomJsonEncoder)
```

Comment:
I think it will all fit on line 331.

Also, I suppose we need to generate a new migration to reflect the `encoder` change on the `parameters` field, do we?